tests/src/core/importType.js

```diff
@@ -224,4 +224,11 @@ describe('importType(name)', function () {
     })
     expect(importType('@test-scope/some-module', foldersContext)).to.equal('external')
   })
+
+  it('`isExternalModule` works with windows directory separator', function() {
+    expect(isExternalModule('foo', {}, 'E:\\path\\to\\node_modules\\foo')).to.equal(true)
+    expect(isExternalModule('foo', {
+      'import/external-module-folders': ['E:\\path\\to\\node_modules'],
```

Comment:
it doesn't seem like we're testing `external-module-folders` here if it was true before? meaning, maybe we need to add a test that returns `false` without `external-module-folders`, and `true` with it?

Reply:
No, without `external-module-folders` still true, because `subpath` defualts to `node_modules`, the test make sure we need replace `subpath`, remove that line will cause error

Reply:
Sorry, to clarify; I'm glad these tests fail without your fix! However, don't we want to also test that a windows path would *not* be external without the right external-module-folders setting?